example/.gitignore

```diff
@@ -13,3 +13,6 @@ packages
 # Ignore it since it just creates noise. When there's a system in place
 # for versioning this project relative to Flutter, remove this.
 pubspec.lock
+
+# Generated by flutter run and flutter build
+Generated.xcconfig
```

Comment:
This should be in a .gitignore in example/macos (especially since the goal is to make those folders relocatable shortly)